quinn-proto/src/lib.rs

```diff
@@ -117,7 +117,7 @@ pub mod fuzzing {
     }
 }
 /// The QUIC protocol version implemented
-const VERSION: u32 = 0xff00_001d;
+const VERSION: u32 = 0xff00_001f;
```

Comment:
Please leave it at the old version - maybe with a comment.
Browsers like Safari and Chrome will now starting to use Chrome, but they will all announce Draft 29 version support even if they are 30+.
With this change things would be needlessly imcompatible.